### ISSUE TITLE
Fusion Crafting injectors not accepting power

### ISSUE THREAD

**Issue by Avnior** [brandon3055/Draconic-Evolution#668]
I'm unable to get any RF into the fusion crafting injectors to do any crafting.

I have checked several things that have been mentioned before. Here is a list of a few:
- Energy I/O Crystal is set to Output (though I tried flipping it between them to make sure I was using the correct one)
- There is power in the I/O Crystal so it should be able to output power.
- I am testing this in a creative world before I do anything in survival

And here are a few extra pieces of info that hopefully will help:
Modpack: FTB Beyond
MC Version: 1.10.2

A few pictures of the setup:
![2017-03-27_17 34 41](https://cloud.githubusercontent.com/assets/26726409/24381003/bffe567c-1313-11e7-9830-b28da7d98665.png)
![2017-03-27_17 34 59](https://cloud.githubusercontent.com/assets/26726409/24381004/c0058c30-1313-11e7-8e47-08ec2faadcb6.png)

**Comment by TheBrain0110** [comment]
Are you sure those IO crystals are getting power? It's showing as only 0.38% full, which is basically empty. They only like to output when they're nearly 100%, and if it was getting power it would be filling up...

**Comment by Avnior** [comment]
I just attached a creative RF storage to it... all of the I/O Crystals are full and none of the Fusion Crafting Injectors are getting any sort of power.

**Comment by Teslachu** [comment]
What do you have the energy i/o crystal on the pylon set to? That one should be set to input. The next question is do you have any power in your core? ~ Also as a note, instead of doing all of the energy i/o crystals on the back of the injectors - you could do a wireless energy crystal and then link it directly to those injectors. I think 1 basic wireless crystal should be able to do up to 16 links/devices.... since you are using the draconic tier stuff - you could probably do the draconic wireless energy crystal and link it to all of your injectors in your screen shots, which would save you mats on all the extra energy i/o crystals and the extra relay. Also, for the max recipe for injectors - you only need 9 (max ever used).... They also need to be the same level as what the recipe calls for... so your 5 draconic injectors and your 4 basic, wouldnt let you do anything above basic level.... Its kind of an all or nothing sort of deal.

**Comment by Avnior** [comment]
A few things that should answer your questions.
1. I do have power in the core
     1a. I have a creative RF power supply linked to the crystals as well so that shouldn't matter anyway.
     1b. I also have the crystal on the creative RF supply set to input as well.
2. I do have the I/O crystal on the pylon set to input
3. All of the I/O crystals have full power in them.
4. Yes I could use the wireless crystals. However, I want to make sure I can use the crystal network for other stuff too.
     4a. Also that doesn't solve the fact that I can't get the I/O crystals to work... It just avoids the problem rather than fixing it.

5. As an extra note, I wasn't sure if the Injectors could be used for lower tiers of crafting so I switched them out to make sure that wasn't the reason why the recipe wasn't working. So thanks for that info. :D

**Comment by Teslachu** [comment]
I am not completely sure about if its really an all or nothing, for the recipe.. but I do know certain recipes do require the various tiers and up to 9. I am trying to recreate your build to figure out what is going on, but it is gonna take quite a bit. XD

**Comment by Teslachu** [comment]
Ok so after a bit of testing - it doesnt like your layout... Idk why but it doesnt. It isn't a problem with the whole different tiers of injectors. I am not sure if you were getting the recipe to even show up, but when I replicated your design in a skyfactory 3 world - it wasnt bringing up the recipe. Upon changing the design to something different, the recipe came up. I have no issues with power or getting it to work that way. 

I have all the I/O crystals as output, same as you. I even used 2 different level injectors... It just did not like the 3x3 pattern for me, for some reason...

Long story short:
* Draconic core - tier 8 with creative power going in on a pylon
* Pylon on the ground, which is set to output (like yours) with an energy i/o crystal.... That crystal is set to input
* That crystal is linked to 1 Draconic Energy Relay Crystal
* That Energy relay crystal is linked to all of the energy i/o crystals (which are all set to output).
* using 2 different lvl injectors for the level up of the wyvern pickaxe (which is fine and worked, cause it only requires the basic one)

Just did a different layout than you.

**Comment by Avnior** [comment]
Thanks for trying this out :D I did have all draconic tier injectors before with the same problem...

**Comment by Avnior** [comment]
I've tried a few different layouts... and none of them have worked... I even tried connecting the injectors using the wireless crystals and that didn't work either. There's still no RF going into the injectors.

**Comment by Teslachu** [comment]
Uh - according to your first screen shot, your pylon is set to in mode... Its blue. Click on that blue ball ontop of it to set it to out mode (orange border on the pylon). That should fix your issue.

**Comment by Teslachu** [comment]
I'm curious if that fixed your issue. ^^

**Comment by Avnior** [comment]
I tried that and it didn't do anything... I also added a creative RF source to completely bypass the energy pylon in case I was doing that wrong.

Here are a few updated shots of what I have now:
Note - Still no RF in the injectors.
![2017-03-29_06 17 06](https://cloud.githubusercontent.com/assets/26726409/24452138/62b550f8-1447-11e7-85b1-d1636f21dc9f.png)
![2017-03-29_06 16 00](https://cloud.githubusercontent.com/assets/26726409/24452137/62b422e6-1447-11e7-9a47-a2e8efa1b37b.png)
![2017-03-29_06 16 52](https://cloud.githubusercontent.com/assets/26726409/24452139/62b6f69c-1447-11e7-9cad-825caf0dd2e0.png)

**Comment by Avnior** [comment]
I tried the wireless crystals to see if that changed anything. (That's why there is the red lasers) and that still didn't get any RF to the injectors.
Note - I did check to make sure I was connecting it directly to the injector and not the I/O crystals.
Note2 - The wireless crystal is connected and receiving power from the relay crystal.

**Comment by Avnior** [comment]
A few quick questions because I am not fully versed in Draconic Evolution...

1. Do the injectors get power before you have the recipe in?
2. When you put the recipe in, do you have to make sure that there is only 1 item in each injector? (because it takes the entire stack that is in my hand)
     2a. If the stack in the injector doesn't affect it, do you still have to have multiple injectors with the item if it requires 2+? (example: the awakened draconium ingots. Could I have multiple in those injectors. Do I need to have both of them there or could I combine them into one injector)
3. Do both sides of injectors have to be the same layout? The same distance from the Fusion Crafting Core? 

I'm asking these clarifying questions to make sure that that is not the reason why my setup isn't working.      

As a separate test, I put a Creative RF source right next to the injector and it still wasn't getting any power either.
![2017-03-29_06 34 04](https://cloud.githubusercontent.com/assets/26726409/24452621/b43016aa-1449-11e7-8b70-12cd08fea197.png)


Oh and one more thing. I didn't change much on the other side. All I did was get rid of a few of them in case the 3x3 was messing something up.
![2017-03-29_06 27 49](https://cloud.githubusercontent.com/assets/26726409/24452429/d5251ffa-1448-11e7-8af8-98c78b32ab97.png)

**Comment by GreatOrator** [comment]
What does the recipe interface look like? but yes, you need one injector for each item of therecipe

**Comment by Avnior** [comment]
Ok... I checked the Fusion Core and it recognized the recipe... (Finally)

Note - the injectors didn't start to receive any power until I started the recipe.
Is that supposed to happen? I know that the Info Tablet said something about a charging stage then the crafting stage... So maybe I misunderstood that and the charging was actually to charge the injectors and not the core.... (or something like that)...

Well either way... It is working now... I'm going to try a few different things just to make sure it is working fully...

(The weird thing is, is the fact that I checked the core and it didn't have the recipe (this morning) then, also this morning, after those messages, I checked and it had the recipe.) :P 

Thanks everyone for looking into this. :D I appreciate the help.

Thanks,
Avnior

**Comment by GreatOrator** [comment]
Yes, it does not charge until you start, the recipes have 2 stages, charging then crafting and the injectors do not store power, only channel it into the recipe.

Glad it got figured out!
